pkg/kubeapiserver/authorizer/reload.go

```diff
@@ -61,6 +61,7 @@ type reloadableAuthorizerResolver struct {
 	nodeAuthorizer *node.NodeAuthorizer
 	rbacAuthorizer *rbac.RBACAuthorizer
 	abacAuthorizer abac.PolicyList
+	compiler       authorizationcel.Compiler
```

Comment:
document where this compiler should be obtained from? is it required to be non-nil?

Reply:
done

Reply:
Note: here we could make it configurable from outside through `Config`. But as this is non-library code of kube-apiserver (by the package name), I have not done that.